tests/integration_tests/flows/test_mysql.py

```diff
@@ -0,0 +1,283 @@
+import unittest
+import csv
+import inspect
+
+import mysql.connector
+
+from mindsdb.utilities.config import Config
+
+from common import (
+    run_environment,
+    get_test_csv,
+    TEST_CONFIG
+)
+
+TEST_CSV = {
+    'name': 'home_rentals.csv',
+    'url': 'https://s3.eu-west-2.amazonaws.com/mindsdb-example-data/home_rentals.csv'
+}
+TEST_DATA_TABLE = 'home_rentals'
+TEST_PREDICTOR_NAME = 'test_predictor'
+
+EXTERNAL_DS_NAME = 'test_external'
+
+config = Config(TEST_CONFIG)
+
+
+def query(q, as_dict=False):
+    con = mysql.connector.connect(
+        host=config['integrations']['default_mysql']['host'],
+        port=config['integrations']['default_mysql']['port'],
+        user=config['integrations']['default_mysql']['user'],
+        passwd=config['integrations']['default_mysql']['password'],
+        db='mindsdb'
+    )
+
+    cur = con.cursor(dictionary=as_dict)
+    cur.execute(q)
+    res = True
```

Comment:
this is strange because I think all tests expect output of `query()` to be an iterable

Reply:
`create database ...` and `create table ...` does not returns iterable.